WORKSPACE

```diff
@@ -207,7 +207,6 @@ new_git_repository(
     name = "operator_framework_community_operators",
     commit = "42131df7167ec0b264c892c1f3c49ba9a72142da",
     remote = "https://github.com/operator-framework/community-operators.git",
-    shallow_since = "1559569397 +0200",
```

Comment:
@fejta @ixdy FYI removing this spits out this comment:

```
DEBUG: Rule 'operator_framework_community_operators' indicated that a canonical reproducible form can be obtained by modifying arguments shallow_since = "1559569397 +0200"
```

However, with it there, in some nondeterministic manner `bazel` will fall over:

```
$ bazel build //prow/...
INFO: Call stack for the definition of repository 'operator_framework_community_operators' which is a new_git_repository (rule definition at /home/skuznets/.cache/bazel/_bazel_skuznets/fe650a89676a24b8d48936a0c5c19473/external/bazel_tools/tools/build_defs/repo/git.bzl:239:22):
 - /home/skuznets/fakego/src/k8s.io/test-infra/WORKSPACE:206:1
ERROR: An error occurred during the fetch of repository 'operator_framework_community_operators':
   Traceback (most recent call last):
	File "/home/skuznets/.cache/bazel/_bazel_skuznets/fe650a89676a24b8d48936a0c5c19473/external/bazel_tools/tools/build_defs/repo/git.bzl", line 227
		_clone_or_update(ctx)
	File "/home/skuznets/.cache/bazel/_bazel_skuznets/fe650a89676a24b8d48936a0c5c19473/external/bazel_tools/tools/build_defs/repo/git.bzl", line 74, in _clone_or_update
		fail(("error cloning %s:\n%s" % (ctx....)))
error cloning operator_framework_community_operators:
+ cd /home/skuznets/.cache/bazel/_bazel_skuznets/fe650a89676a24b8d48936a0c5c19473/external
+ rm -rf /home/skuznets/.cache/bazel/_bazel_skuznets/fe650a89676a24b8d48936a0c5c19473/external/operator_framework_community_operators /home/skuznets/.cache/bazel/_bazel_skuznets/fe650a89676a24b8d48936a0c5c19473/external/operator_framework_community_operators
+ git clone '--shallow-since=1559569397 +0200' https://github.com/operator-framework/community-operators.git /home/skuznets/.cache/bazel/_bazel_skuznets/fe650a89676a24b8d48936a0c5c19473/external/operator_framework_community_operators
Cloning into '/home/skuznets/.cache/bazel/_bazel_skuznets/fe650a89676a24b8d48936a0c5c19473/external/operator_framework_community_operators'...
+ git -C /home/skuznets/.cache/bazel/_bazel_skuznets/fe650a89676a24b8d48936a0c5c19473/external/operator_framework_community_operators reset --hard 42131df7167ec0b264c892c1f3c49ba9a72142da
fatal: Could not parse object '42131df7167ec0b264c892c1f3c49ba9a72142da'.
+ git -C /home/skuznets/.cache/bazel/_bazel_skuznets/fe650a89676a24b8d48936a0c5c19473/external/operator_framework_community_operators fetch '--shallow-since=1559569397 +0200' origin 42131df7167ec0b264c892c1f3c49ba9a72142da:42131df7167ec0b264c892c1f3c49ba9a72142da
error: Server does not allow request for unadvertised object 42131df7167ec0b264c892c1f3c49ba9a72142da
+ git -C /home/skuznets/.cache/bazel/_bazel_skuznets/fe650a89676a24b8d48936a0c5c19473/external/operator_framework_community_operators fetch origin 42131df7167ec0b264c892c1f3c49ba9a72142da:42131df7167ec0b264c892c1f3c49ba9a72142da
error: Server does not allow request for unadvertised object 42131df7167ec0b264c892c1f3c49ba9a72142da
ERROR: /home/skuznets/fakego/src/k8s.io/test-infra/prow/monitoring/BUILD.bazel:19:1: no such package '@operator_framework_community_operators//': Traceback (most recent call last):
	File "/home/skuznets/.cache/bazel/_bazel_skuznets/fe650a89676a24b8d48936a0c5c19473/external/bazel_tools/tools/build_defs/repo/git.bzl", line 227
		_clone_or_update(ctx)
	File "/home/skuznets/.cache/bazel/_bazel_skuznets/fe650a89676a24b8d48936a0c5c19473/external/bazel_tools/tools/build_defs/repo/git.bzl", line 74, in _clone_or_update
		fail(("error cloning %s:\n%s" % (ctx....)))
error cloning operator_framework_community_operators:
+ cd /home/skuznets/.cache/bazel/_bazel_skuznets/fe650a89676a24b8d48936a0c5c19473/external
+ rm -rf /home/skuznets/.cache/bazel/_bazel_skuznets/fe650a89676a24b8d48936a0c5c19473/external/operator_framework_community_operators /home/skuznets/.cache/bazel/_bazel_skuznets/fe650a89676a24b8d48936a0c5c19473/external/operator_framework_community_operators
+ git clone '--shallow-since=1559569397 +0200' https://github.com/operator-framework/community-operators.git /home/skuznets/.cache/bazel/_bazel_skuznets/fe650a89676a24b8d48936a0c5c19473/external/operator_framework_community_operators
Cloning into '/home/skuznets/.cache/bazel/_bazel_skuznets/fe650a89676a24b8d48936a0c5c19473/external/operator_framework_community_operators'...
+ git -C /home/skuznets/.cache/bazel/_bazel_skuznets/fe650a89676a24b8d48936a0c5c19473/external/operator_framework_community_operators reset --hard 42131df7167ec0b264c892c1f3c49ba9a72142da
fatal: Could not parse object '42131df7167ec0b264c892c1f3c49ba9a72142da'.
+ git -C /home/skuznets/.cache/bazel/_bazel_skuznets/fe650a89676a24b8d48936a0c5c19473/external/operator_framework_community_operators fetch '--shallow-since=1559569397 +0200' origin 42131df7167ec0b264c892c1f3c49ba9a72142da:42131df7167ec0b264c892c1f3c49ba9a72142da
error: Server does not allow request for unadvertised object 42131df7167ec0b264c892c1f3c49ba9a72142da
+ git -C /home/skuznets/.cache/bazel/_bazel_skuznets/fe650a89676a24b8d48936a0c5c19473/external/operator_framework_community_operators fetch origin 42131df7167ec0b264c892c1f3c49ba9a72142da:42131df7167ec0b264c892c1f3c49ba9a72142da
error: Server does not allow request for unadvertised object 42131df7167ec0b264c892c1f3c49ba9a72142da
 and referenced by '//prow/monitoring:alertmanager_crd'
ERROR: Analysis of target '//prow/monitoring:prometheus_operator_crd.create' failed; build aborted: Analysis failed
```

The directory with the shallow clone just gets three recent commits:

```
$ git -C /home/skuznets/.cache/bazel/_bazel_skuznets/fe650a89676a24b8d48936a0c5c19473/external/operator_framework_community_operators log --oneline
ff73320 (HEAD -> master, origin/master, origin/HEAD) Merge pull request #419 from jcrossley3/allns
d2f1956 Fix example CR and support only AllNamespaces installmode
e131d8a (grafted) Merge pull request #400 from raffaelespazzoli/cert-utils-operator
```

This might be a `git` bug? In any case ... removing it makes things build

Reply:
Removed.